src/utils/metrics.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  PoolMetricsResponse,
+  PoolMetrics,
+  PoolMetric,
+} from "../types/PoolMetrics";
+
+export const processPoolMetrics = (
+  metrics: PoolMetricsResponse
+): PoolMetrics => {
+  const result = {} as PoolMetrics;
+  for (let metricKey in metrics) {
```

Comment:
I was just wondering if this for would work like this (it seems simpler):
```suggestion
  for (let metricKey of metrics) {
        result[metricKey] = [];
        
        if (metricKey) {
          result[metricKey] = [
            {
              poolLiquidity: metricKey.poolLiquidity,
              poolTotalSwapFee: metricKey.poolTotalSwapFee,
              poolTotalSwapVolume: metricKey.poolTotalSwapVolume,
              price:
                metricKey.tokenAmountIn /
                metricKey.tokenAmountOut,
            }
          ];
        }
      }
```

Reply:
I don't think this works because the raw data comes in as an object so we're iterating through the keys 
<img width="756" alt="Screenshot 2021-04-23 at 11 28 47" src="https://user-images.githubusercontent.com/629600/115860887-0be3eb00-a42a-11eb-890b-8a0d6bb2f505.png">

Reply:
aahh sorry, you are right.
`for (const [key, value] of Object.entries(metrics)) {...}`
This should work (I hope :pray: )

Reply:
👍